keepvariable/keepvariable_core.py

```diff
@@ -522,12 +545,14 @@ def set(self, key: str, value: str, additional_params: Optional[dict] = None):
         return result
 
     def get(self, key: str) -> Optional[Any]:
-        value = self.redis.get(key)
-
-        if value is None:
-            return value
-
-        decoded_value = self.decode_loaded_value(value)
+        try:
+            value = self.redis.get(key)
+            if value is None:
+                return value
+            decoded_value = self.decode_loaded_value(value)
```

Comment:
could `value is None` be handled in `self.decode_loaded_value`?

Reply:
No @Omastto1 that doesnt make sense, decode works with "not None" values which are usually dicts. None can be result here even for not-existing key - which doesnt need to be decoded at all, @ZaiZu2 please add comment to that line of code